src/main/java/com/tileman/TilemanModePlugin.java

```diff
@@ -131,6 +131,14 @@ TilemanModeConfig provideConfig(ConfigManager configManager) {
     public void onMenuOptionClicked(MenuOptionClicked event) {
         if (event.getMenuAction().getId() != MenuAction.RUNELITE.getId() ||
                 !(event.getMenuOption().equals(MARK) || event.getMenuOption().equals(UNMARK))) {
+            if(event.getMenuAction() == MenuAction.WALK && !modeTilescontainSelectedSceneTile() ){
+                //We are attempting to walking to a location and it is not in our list
+                if(!config.automarkTiles() || (config.automarkTiles() && remainingTiles == 0)) {
```

Comment:
I'm thinking this functionality should be optional, could you add a config option?